electron/js/preload.js

```diff
@@ -65,6 +65,24 @@ const addDragRegion = () => {
     const titleBar = document.createElement('div');
     titleBar.className = 'drag-region';
     document.body.appendChild(titleBar);
+
+    let isDragging = false;
+    let isMoved = false;
+
+    titleBar.addEventListener('mousedown', () => (isDragging = true));
+    document.addEventListener('mousemove', () => (isMoved = isDragging));
+
+    titleBar.addEventListener('mouseup', e => {
```

Comment:
Characters are free. We still don't like non descriptive variable names even if it is `e`